docs/UserGuide.md

```diff
@@ -122,6 +122,28 @@ To edit name and address of your tutee:
 Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the list.
 
+### Marking a person as paid : `paid`
+
+**Description** Mark the specific tutee as paid in the list.
+
+**Format**: `paid INDEX`
+
+**Expected Input**:
+
+* **Index (Compulsory Field)**: Numbers between 1 to the number of people inside the list.
+
+**Expected Output when the command succeeds**: MARK PERSON PAID SUCCESS, Paid: true
+
+**Expected Output when the command fails**: Invalid command format! paidExample: paid 1
+
+Examples:
+* `list` followed by `paid 2` marks the 2nd person in the list.
```

Comment:
Nice example! We could start to implement visual aids such as screenshots for examples like this in the future (across the UG)